### ISSUE TITLE
Overview types Sourcodes available

### ISSUE THREAD

**Issue by ElsGear** [clough42/electronic-leadscrew#64]
Hi James, firstly thanks for all work you've made open source.
All is well documented but I have a hard time overviewing the types of source codes available and where to download.
* I understand that there will be just 1 leading master code with several optional upgrades.
* But what is the MultiConfigBuild update relative to the master code.
* Also useful code "Leadscrew speed warnings, hook up power button" from alexphredorg is how to be implemented to your newest version of the master code for not missing your updates.

In short, could you update your leading master code to implement more of the coded helpful functions without slowing down the gear translation main function of the ELS-controler?

Thanks...

**Comment by clough42** [comment]
I have merged several of the PRs that people have submitted.  The power button is now active on the dev branch and will be in release 1.3.01, coming soon.  I have not yet implemented speed warnings.
